source/includes/steps-starting-compass.yaml

```diff
@@ -15,7 +15,13 @@ content: |
        * - :guilabel:`Hostname`
 
          - The hostname of the machine where the ``mongod`` instance is
-           running.
+           running. 
+
+           .. tip::
+
+              Compass automatically populates other fields in this dialog
```

Comment:
?
If you enter a MongoDB connection string in this field, Compass automatically parses the string to populate relevant field in this dialog.